recipes/tbb/oneTBB/conanfile.py

```diff
@@ -0,0 +1,105 @@
+import os, shutil
+from conan.tools.cmake import CMake, CMakeToolchain
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.39.0"
+
+class ConanFile(ConanFile):
```

Comment:
```suggestion
class TBBConan(ConanFile):
```

Reply:
Why?

Reply:
It's weird to name your subclass the same as your superclass + this is similar to all other recipes.

Reply:
Functionally there is no difference, and practically a common name (such as `ConanFile`) is more useful as you don't have to change it every time you copy and paste.